.azure-pipelines/util/analyze-steps.yml

```diff
@@ -15,14 +15,14 @@ steps:
   inputs:
     command: custom
     custom: msbuild
-    arguments: 'build.proj /t:GenerateHelp /p:Configuration=${{ parameters.configuration }}'
+    arguments: 'build.proj /t:GenerateHelp /p:Configuration=${{ parameters.configuration }};PullRequestNumber=$(System.PullRequest.PullRequestNumber)'
```

Comment:
Could you verify this works on non-PullRequest builds? Put these changes to a branch in Azure, and then queue the build in ADO manually, targeting that branch.

Reply:
Manual build here: https://dev.azure.com/azure-sdk/public/_build/results?buildId=38001